### DISCUSSION TITLE
Showing Github stars next to the open-source options

### DISCUSSION THREAD

**Discussion by sureshjoshi** [tycrek/degoogle#418] [Ideas]
I think I've seen this done in other repos (awesome-xyz kinda repos), but I think it'd be visually handy to see the popularity of certain projects. 

Not sure if it's straightforward though

**Comment by sureshjoshi** [comment]
Oh, and sorry, first thing I should have said... I just discovered this repo earlier today and I think it's excellent!

**Comment by tycrek** [comment]
This is a really cool idea, so I just wrote it to the build script and will publish it shortly :D

**Comment by tycrek** [comment]
Here's what it looks like live

![stars](https://ass.rip/Vg2at6byTTiS/direct)

**Reply by sureshjoshi** [reply]
Cool! Thanks @tycrek!

Personally, I find that much easier to scroll through. In a lot of aggregation repos, they can be littered with a bunch of very small or unmaintained repos, but when you see a couple K stars - it gives a nice starting point

**Comment by toastal** [comment]
This is a bad idea. Trying to de-Google while supporting Microsoft GitHub's idea of a "social" code platform. This is just shifting from one US megacorp to another.

**Comment by sureshjoshi** [comment]
My next statement was about to be "well, then this project shouldn't even be on Github :)" - but then I saw you created this discussion about that, so I'll link to it here: https://github.com/tycrek/degoogle/discussions/446  

If this project is already on Github, and Github has stars which tend to have some reasonable correlation with relevancy, activity, and overall awareness - they may as well be shown on this repo. 

However, if the issue is fundamentally with not supporting Microsoft and not wanting to use Github, then your https://github.com/tycrek/degoogle/discussions/446 is a better forum for that.

**Comment by tycrek** [comment]
I agree that further discussion can be had in #446 as the original purpose of this one has been resolved.
